src/pkg/packager/deploy.go

```diff
@@ -304,10 +304,26 @@ func (p *Packager) getUpdatedValueTemplate(component types.ZarfComponent) (value
 	defer spinner.Stop()
 
 	state, err := p.cluster.LoadZarfState()
+	// Return on error if we are not in YOLO mode
```

Comment:
I thought this block looked a little weird, wanted to suggest this (but won't block the PR on it):

``` Go
	// Return an error if we are unable to load state (assuming we are not in YOLO mode)
	if (err != nil && !p.cfg.Pkg.Metadata.YOLO) || (state.Distro == "" && !p.cfg.Pkg.Metadata.YOLO) {
		return values, fmt.Errorf("unable to load the Zarf State from the Kubernetes cluster: %w", err)
	}

	if p.cfg.Pkg.Metadata.YOLO {
		state.Distro = "YOLO"
		message.Warn("This package is in YOLO mode, ...")
	}
```